lib/ansible/modules/cloud/docker/docker_network.py

```diff
@@ -249,20 +260,26 @@ def has_different_config(self, net):
                 different = True
                 differences.append('ipam_options')
             else:
-                for key, value in self.parameters.ipam_options.items():
```

Comment:
Is there a reason for this change? Is it needed when IPv6 is enabled (does `net['IPAM']['Config']` then have two entries, one for IPv4 and one for IPv6)?